src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String CSV_SPLIT_BY = ",";
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        String line;
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter bw = new BufferedWriter(new FileWriter(toFileName))) {
+            while ((line = br.readLine()) != null) {
+                String[] data = line.split(CSV_SPLIT_BY);
+                String operation = data[0].trim();
+                int amount = Integer.parseInt(data[1].trim());
+                if (operation.equals("supply")) {
+                    supplyAmount += amount;
+                } else if (operation.equals("buy")) {
+                    buyAmount += amount;
+                }
+            }
+            int result = supplyAmount - buyAmount;
+            bw.write("supply," + supplyAmount);
+            bw.newLine();
+            bw.write("buy," + buyAmount);
+            bw.newLine();
+            bw.write("result," + result);
```

Comment:
```suggestion
```

It is necessary to write not one line at a time, but a fully formed report to a file